tips/TIP-0018/tip-0018.md

```diff
@@ -0,0 +1,3832 @@
+                        <td>ByteArray[32]</td>
+                        <td>The raw bytes of the <i>Alias ID</i> which is the BLAKE2b-256 hash of the outputID that created it.</td>
+                    </tr>
+                </table>
+            </details>
+            <details>
+                <summary>NFT Address</summary>
+                <table>
+                    <tr>
+                        <td><b>Name</b></td>
+                        <td><b>Type</b></td>
+                        <td><b>Description</b></td>
+                    </tr>
+                    <tr>
+                        <td>Address Type</td>
+                        <td>uint8</td>
+                        <td>
+                            Set to <strong>value 16</strong> to denote an <i>NFT Address</i>.
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>NFT ID</td>
+                        <td>ByteArray[32]</td>
+                        <td>The raw bytes of the <i>NFT ID</i> which is the BLAKE2b-256 hash of the outputID that created it.</td>
+                    </tr>
+                </table>
+            </details>
+        </td>
+    </tr>
+</table>
+
+| :information_source:  Good to know about address format |
+|---------------------------------------------------------|
+
+The <i>Address Type</i> byte of a raw address has an effect on the starting character of the bech32 encoded address,
+which is the recommended address format for user facing applications.
+
+A usual bech32 encoded mainnet address starts with `iota1`, and continues with the bech32 encoded bytes of the address.
+By choosing <i>Address Type</i> as a multiple of 8 for different address types, the first character after the `1`
+separator in the bech32 address will always be different.
+
+| Address | Type Byte as `uint8` | Bech32 Encoded |
+|---------|----------------------|----------------|
+| Ed25519 | 0                    | iota1**q**...  |
+| Alias   | 8                    | iota1**p**...  |
+| NFT     | 16                   | iota1**z**...  |
+
+A user can identify by looking at the address whether it is a signature backed address, a smart contract chain account
+or an NFT address.
+
+##### Storage Deposit Return Unlock Condition
+
+This unlock condition is employed to achieve conditional sending. An output that has
+<i>Storage Deposit Return Unlock Condition</i> specified can only be consumed in a transaction that deposits
+`Return Amount` IOTA coins into `Return Address`. When several of such outputs are consumed, their return amounts per
+`Return Addresses` are summed up and the output side must deposit this total sum per `Return Address`.
+
+###### Additional syntactic transaction validation rule:
+
+- `Minimum Storage Deposit` is the storage deposit in the base currency required for a <i>Basic Output</i> that only
+  has an <i>Address Unlock Condition</i> with `Return Address`, no additional unlock conditions, no features and
```

Comment:
```suggestion
  has an <i>Address Unlock Condition</i>, no additional unlock conditions, no features and
```

Reply:
`Return Address` here refers to the field of SDRUC of the consumed output, so I don't get why this should be removed.

Reply:
To me it sounds just like a definition for `Minimum Storage Deposit`, which is unrelated to the SDRUC

Reply:
Alright, if you say so :cat: